README.md

```diff
@@ -299,13 +303,46 @@ laptop asset should be converted to a folder, and the license moved into it.
   Pass `git-command-args` as arguments to `git`, using the Onyo repository as
   the git repository. This is most valuable when used in conjunction with
   `ONYO_REPOSITORY_DIR`.
-
+- `onyo new [--non-interactive, -I] directory`:
+
+  Creates a new `asset` in `directory`. The command opens a dialog that asks for
+  the field names defined by the asset name scheme, and after creation opens the
+  new `asset` file with the editor.
+  After the editing is done, the new file will be checked for the validity of
+  it's YAML syntax.
+
+  - `--non-interactive` : Suppress opening of editor after file creation.
```

Comment:
I'd like to clarify something here:
- `onyo new` will spawn an interactive prompt, and the user will enter information. It will then open an editor, for the remaining information to be entered.
- `onyo new --non-interactive` will spawn an interactive prompt, and the user will enter information. It will not open an editor.
- 
If this is the behavior, then at minimum `--non-interactive` is poorly named, as it still requires interaction.

To me, the utility of a feature like `--non-interactive` is to allow scripts to use it. How would one script a call to `onyo new` with sufficient information to not require interaction?